kinder/ci/tools/update-workflows/templates/testinfra/kubeadm-kinder-control-plane-local-kubelet-mode.yaml

```diff
@@ -9,7 +9,7 @@
     testgrid-dashboards: sig-cluster-lifecycle-kubeadm
     testgrid-tab-name: kubeadm-kinder-control-plane-kubelet-local-mode-{{ dashVer .KubernetesVersion }}
     testgrid-alert-email: sig-cluster-lifecycle-kubeadm-alerts@kubernetes.io
-    description: "OWNER: sig-cluster-lifecycle (kinder); Uses kubeadm/kinder to create a cluster using the ControlPlaneKubeletLocalMode feature gate and run the kubeadm-e2e and the conformance suites"
+    description: "OWNER: sig-cluster-lifecycle (kinder); Uses kubeadm/kinder to create a cluster using the ControlPlaneKubeletLocalMode feature gate disabled and run the kubeadm-e2e and the conformance suites"
```

Comment:
we can skip the test infra update, i think. one less pr and it just matters for us the maintainers whether the fg is enabled or disabled.